### PR TITLE
Typos / minor errors

### PR DESCRIPTION
**Pull request by wturrell**
(Btw I'm loving this book, it's fascinating.)

### CODE REVIEW

File: Content/Part 2/8-support-and-documentation.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ In [December 2003 the WordPress wiki launched](http://wordpress.org/news/2003/12
 
 Originally, the wiki was designed to complement the [official documentation](http://web.archive.org/web/20030811221523/http://wordpress.org/docs/). The [landing page informed visitors](http://web.archive.org/web/20031224140754/http://wiki.wordpress.org/) that it was "designed for us to be able to work together on projects." While developers worked towards shipping WordPress 1.0 in January 2004, other community members worked furiously on the wiki. 
 
-This was in contrast to the aborted work on the official documentation. An [FAQ](https://web.archive.org/web/20040402000122/http://wordpress.org/docs/faq/) and [template documentation](https://web.archive.org/web/20040411104706/http://wordpress.org/docs/template/) were created. But the [majority of documentation was written on the wiki](https://web.archive.org/web/20040323105321/http://wiki.wordpress.org/).
```
Comment (smckeown) [inline Content/Part 2/8-support-and-documentation.md]: "An FAQ" is correct - The use of A/An before FAQ depends on pronunciation. There's an interesting discussion of it here: http://english.stackexchange.com/questions/1016/do-you-use-a-or-an-before-acronyms

**Comment by smckeown** [comment]
Thanks for the PR. Glad you're enjoying the book :) Can you remove the edit to "an FAQ" before I merge?

**Comment by wturrell** [comment]
@smckeown done.

**Comment by DrewAPicture** [comment]
@wturrell would you care to refresh your PR?

**Comment by wturrell** [comment]
@DrewAPicture done (I hope…)

**Comment by DrewAPicture** [comment]
Looks good. Thanks for the fixes!

**Pull request merged**